<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IDH9V1V" failures="1" tests="2" name="Test-Runner3" time="45.272" errors="0" timestamp="2024-01-09T08:13:45 IST">
  <testcase classname="contacts.CreateContactWithLeadSource1" name="createContactWithLeadSource1" time="20.113"/>
  <testcase classname="contacts.CreateContactWithLeadSource1" name="demoTest" time="0.01">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contacts.CreateContactWithLeadSource1.demoTest(CreateContactWithLeadSource1.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- demoTest -->
</testsuite> <!-- Test-Runner3 -->
